pom.xml

```diff
@@ -1126,6 +1126,20 @@
                 <artifactId>failsafe</artifactId>
                 <version>2.0.1</version>
             </dependency>
+
+            <dependency>
+                <groupId>org.mockito</groupId>
```

Comment:
We have found over the years that mocking frameworks lead to brittle tests that really don't end up testing much.  I know there are good ways to use mocking frameworks, but it is too easy to use them in ways that ends up with difficult to maintain tests.  As a result, we have actively avoided them in Presto.  

Instead of mocking frameworks, we typically have a local, in-memory, version of interfaces or services for testing.  For example, TestingAccessControlManager or TestingMetadata.  I'll comment below on how I think we can test this without introducing mockito to Presto.